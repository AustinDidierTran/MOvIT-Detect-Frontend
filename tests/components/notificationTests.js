import React from 'react';
import TestUtils from 'react-dom/test-utils';
import PropTypes from 'prop-types';
import GoalProgress from '../../src/components/notification';

describe('Notification Tests', () => {
  it('should have proptypes', () => {
    // Actual value
    const actualValue = GoalProgress.WrappedComponent.propTypes;

    // Expected value
    const expectedValue = {
      language: PropTypes.string.isRequired,
      header: PropTypes.object, // eslint-disable-line react/forbid-prop-types
    };

    // Test
    expect(JSON.stringify(actualValue)).toEqual(JSON.stringify(expectedValue));
  });
});
