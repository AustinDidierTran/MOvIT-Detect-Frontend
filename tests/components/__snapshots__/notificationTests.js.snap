// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification Tests should match the snapshot 1`] = `
<div
  className="row ml-2 mt-5"
>
  <div
    className="mb-2 mr-3"
  >
    <button
      className="btn btn-lg"
      id="calibrate-button"
      onClick={[Function]}
    >
      calibrate.en
    </button>
  </div>
  <div
    className="mr-3 mb-2"
  >
    <button
      className="btn btn-lg"
      id="turn-on-button"
      onClick={[Function]}
    >
      alert.on.en
    </button>
  </div>
  <div
    className="mr-3 mb-2"
  >
    <button
      className="btn btn-lg"
      id="turn-off-button"
      onClick={[Function]}
    >
      alert.off.en
    </button>
  </div>
  <Connect(Countdown)
    onComplete={[Function]}
    time={10}
    title="calibrating.en"
  />
</div>
`;
